Formula/j/jr.rb

```diff
@@ -0,0 +1,22 @@
+class Jr < Formula
+  desc "CLI program that helps you to create quality random data for your applications"
+  homepage "https://jrnd.io/"
+  url "https://github.com/ugol/jr/archive/refs/tags/v0.3.3.tar.gz"
+  sha256 "fa60365c0ca7b5ff70ef357ff362c7da069aa07a5daa8303f0af04ae75d04f67"
+  license "MIT"
+  head "https://github.com/ugol/jr.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    libexec.install Dir["build/*"]
+    prefix.install "config/jrconfig.json"
+    prefix.install "templates"
```

Comment:
Can these not go into the usual `etc` directories for configuration and `share` for application files?

Reply:
I think I can put both under `etc`.

Reply:
But templates aren't config so they should go to `share`

Reply:
The problem is that I can set just one `JR_HOME` and the tool is supposed to find under its home the needed files. I cannot split in two folders as for now.